extensions/blocks/simple-payments/simple-payments.php

```diff
@@ -88,7 +88,7 @@ function render_block( $attr, $content ) {
 function amp_skip_post( $skip, $post_id, $post ) {
 	// When AMP is on standard mode, there are no non-AMP posts to link to where the purchase can be completed, so let's
 	// prevent the post from being available in AMP.
-	if ( amp_is_canonical() && has_block( 'jetpack/simple-payments', $post->post_content ) ) {
+	if ( function_exists( 'amp_is_canonical' ) && amp_is_canonical() && has_block( 'jetpack/simple-payments', $post->post_content ) ) {
```

Comment:
Should we be extra safe with this?

```suggestion
	if ( function_exists( 'amp_is_canonical' ) && \amp_is_canonical() && has_block( 'jetpack/simple-payments', $post->post_content ) ) {
```